### ISSUE TITLE
Init for Kotlin and Groovy lib/app respectively use outdated Kotlin and Groovy version

### ISSUE THREAD

**Issue by pkubowicz** [gradle/gradle#7793]
Running 'gradle init' to create an empty project makes the new project use a Kotlin version older than the one used by Gradle.

### Expected Behavior
Generated project and Gradle use the same Kotlin version.

### Current Behavior
Gradle:
```
------------------------------------------------------------
Gradle 5.0-rc-4
------------------------------------------------------------

Build time:   2018-11-20 09:26:42 UTC
Revision:     50c28fc7284c0aa377c8edb49723d9258565780f

Kotlin DSL:   1.0.4
Kotlin:       1.3.10
```
New project's `build.gradle.kts`:
```
id("org.jetbrains.kotlin.jvm").version("1.2.71")
```

### Steps to Reproduce (for bugs)
Update to gradle-5.0-rc-4 and run:
```
./gradlew init --type=kotlin-application --dsl kotlin
```

### Your Environment
 * Build scan URL: https://scans.gradle.com/s/tyrayoysrs6zy

**Comment by eskatos** [comment]
Thanks for the report @pkubowicz!
This will be fixed in 5.0-RC5, see https://github.com/gradle/gradle/pull/7795

**Comment by pkubowicz** [comment]
You can consider doing the same for Groovy: use `'org.codehaus.groovy:groovy-all:2.5.4'` and `'org.spockframework:spock-core:1.2-groovy-2.5'`. Gradle 5.0-RC4 uses Groovy 2.5.4 but generates 'groovy-application' using Groovy 2.4.15.

**Comment by eskatos** [comment]
@pkubowicz, right, thanks! See #7796

**Comment by eskatos** [comment]
Fixed in #7795 and #7796, closing
